### ISSUE TITLE
Build failure: zeek 6.0.2 on Darwin aarch64

### ISSUE THREAD

**Issue by osnyx** [NixOS/nixpkgs#289960]
### Steps To Reproduce

1. build `zeek`

### Build log

```
[ 70%] Built target btest-peers
[ 70%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/data.cc.o
[ 71%] Building CXX object doc/_examples/CMakeFiles/synopsis.dir/synopsis.cc.o
[ 72%] Building CXX object doc/_examples/CMakeFiles/comm.dir/comm.cc.o
[ 73%] Building CXX object doc/_examples/CMakeFiles/stores.dir/stores.cc.o
[ 73%] Building CXX object doc/_examples/CMakeFiles/ping.dir/ping.cc.o
[ 74%] Linking CXX executable ../../bin/synopsis
[ 74%] Built target synopsis
[ 75%] Building CXX object doc/_examples/CMakeFiles/pong.dir/pong.cc.o
[ 76%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/detail/peer_status_map.cc.o
[ 77%] Linking CXX executable ../../bin/comm
[ 78%] Linking CXX executable ../../bin/stores
[ 78%] Built target comm
[ 79%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/domain_options.cc.o
[ 79%] Built target stores
[ 80%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/error.cc.o
[ 80%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/filter_type.cc.o
[ 81%] Linking CXX executable ../../bin/ping
[ 81%] Built target ping
[ 82%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/internal/channel.cc.o
[ 83%] Linking CXX executable ../../bin/pong
[ 83%] Built target pong
[ 84%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/internal/core_actor.cc.o
[ 85%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/internal/json_type_mapper.cc.o
[ 86%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/internal/metric_collector.cc.o
[ 86%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/internal/metric_exporter.cc.o
[ 87%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/master.cc.o
[ 88%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/publisher.cc.o
[ 89%] Linking CXX shared module ../../python/broker/_broker.so
/nix/store/fjw794m67an9sm25vghjgms23nmhw06q-cctools-binutils-darwin-16.0.6-973.0.1/bin/strip: error: unknown argument '-u'
make[2]: *** [bindings/python/CMakeFiles/_broker.dir/build.make:168: python/broker/_broker.so] Error 1
make[2]: *** Deleting file 'python/broker/_broker.so'
make[1]: *** [CMakeFiles/Makefile2:297: bindings/python/CMakeFiles/_broker.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 90%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/radix_tree.cc.o
[ 90%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/status.cc.o
[ 91%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/status_subscriber.cc.o
[ 92%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/store.cc.o
[ 93%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/store_event.cc.o
[ 93%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/subscriber.cc.o
[ 94%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/system/peering.cc.o
[ 95%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/system/shutdown.cc.o
[ 96%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/telemetry/histogram.cc.o
[ 96%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/test.cc.o
[ 97%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/topic.cc.o
[ 98%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/zeek.cc.o
[100%] Linking CXX executable ../bin/broker-test
[100%] Built target broker-test
make: *** [Makefile:146: all] Error 2
error: builder for '/nix/store/c9b6wp078bhqjb3fzx2dq5dbgvy5a3cj-zeek-broker-unstable-2023-02-01.drv' failed with exit code 2;
       last 10 log lines:
       > [ 93%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/subscriber.cc.o
       > [ 94%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/system/peering.cc.o
       > [ 95%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/system/shutdown.cc.o
       > [ 96%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/telemetry/histogram.cc.o
       > [ 96%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/test.cc.o
       > [ 97%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/topic.cc.o
       > [ 98%] Building CXX object tests/CMakeFiles/broker-test.dir/cpp/zeek.cc.o
       > [100%] Linking CXX executable ../bin/broker-test
       > [100%] Built target broker-test
       > make: *** [Makefile:146: all] Error 2
       For full logs, run 'nix-store -l /nix/store/c9b6wp078bhqjb3fzx2dq5dbgvy5a3cj-zeek-broker-unstable-2023-02-01.drv'.
error: 1 dependencies of derivation '/nix/store/am5b4ismiqskh4pb4prq2fxi5zblgszz-zeek-6.0.2.drv' failed to build
```

### Notify maintainers
@pSub @marsam @tobim 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 23.3.0, macOS 14.3.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.1`
 - channels(root): `"nixpkgs"`
 - channels(os): `"NUR, darwin, home-manager-23.11.tar.gz, nixpkgs-23.11-darwin"`
 - nixpkgs: `/Users/os/.nix-defexpr/channels/nixpkgs`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by emilytrau** [comment]
`zeek` has been patched to build on darwin in #277538, available in `nixpkgs-unstable`. As 6.1.0 is a breaking release we can't port this change automatically. I would recommend using the current zeek release from the unstable channel at least until the next stable nixpkgs release 24.05. If this isn't possible and zeek 6.0.2 is required please let us know and we can backport this change to nixpkgs 23.11

**Comment by osnyx** [comment]
That's fine for me, thanks.
